reproserver/web/templates/setup.html

```diff
@@ -19,41 +20,43 @@ <h1>Package {{ filename | truncate(60) }}</h1>
 
 <h2>Run the experiment</h2>
 
-<form method="POST" action="{{ reverse_url('start_run', upload_short_id) }}" enctype="multipart/form-data">
+<form method="POST" enctype="multipart/form-data">
```

Comment:
`action` can probably stay here

Reply:
wouldn't it submit the form twice, because I'm sending an XHR on button click

Reply:
It is still set, you moved it to `formaction` on the button. I don't think that is necessary?

Reply:
don't need this anymore, this change happened due to previous design, where we have two buttons reproduce/record in the same form. will submit changes in the next commit